--- Nestopia/Makefile.orig	2008-06-26 06:09:02.000000000 +0200
+++ Nestopia/Makefile	2008-07-15 14:04:05.000000000 +0200
@@ -5,7 +5,8 @@
 
 CC   = gcc
 CPP  = g++
-CFLAGS = -c -O3 -g3
+RPMFLAGS = 
+CFLAGS = ${RPMFLAGS}
 CFLAGS += -DNST_PRAGMA_ONCE_SUPPORT -D_SZ_ONE_DIRECTORY
 CFLAGS += -Isource -Isource/core -Isource/zlib -Isource/core/api -Isource/core/board -Isource/core/input -Isource/linux/unzip 
 CFLAGS += -Isource/core/vssystem -Isource/linux -Isource/nes_ntsc -I.. -I../nes_ntsc -Isource/linux/7zip 
@@ -18,7 +19,7 @@
 CPPFLAGS = -Wno-deprecated -fno-rtti
 
 EXE  = nst
-LIBS = -lm -lz -lasound  `sdl-config --libs` `pkg-config --libs gtk+-2.0`
+LIBS = -lm -lz -lasound  -lnes_ntsc `sdl-config --libs` `pkg-config --libs gtk+-2.0`
 
 # OpenGL Support
 CFLAGS += -DINCLUDE_OPENGL
@@ -147,15 +148,15 @@
 # build rules
 objs/%.o: source/%.c
 	@echo Compiling $<...
-	@$(CC) $(CFLAGS) $< -o $@
+	$(CC) $(CFLAGS) $< -o $@
 
 objs/%.o: source/%.cpp
 	@echo Compiling $<...
-	@$(CC) $(CFLAGS) $(CPPFLAGS) $< -o $@
+	$(CC) $(CFLAGS) $(CPPFLAGS) $< -o $@
 
 objs/%.o: source/%.o
 	@echo Compiling $<...
-	@$.o) $(CFLAGS) $.oFLAGS) $< -o $@
+	$.o) $(CFLAGS) $.oFLAGS) $< -o $@
 
 all: maketree $(EXE) $(GENNSTCONTROLS)
 
@@ -168,7 +169,7 @@
 # link the commandline exe
 $(EXE): $(OBJS)
 	@echo Linking $@...
-	@$(CPP) -g -o $(EXE) $^ $(LIBS)
+	$(CPP) -g -o $(EXE) $^ $(LIBS)
 
 clean:
 	-@rm -f $(OBJS) $(EXE) $(GENNSTCONTROLS)
