diff -durN nestopia-1.51.0.orig/Makefile.am nestopia-1.51.0/Makefile.am
--- nestopia-1.51.0.orig/Makefile.am	2021-05-19 22:42:36.000000000 +0200
+++ nestopia-1.51.0/Makefile.am	2021-05-24 12:13:04.369985601 +0200
@@ -8,6 +8,7 @@
 nestopia_CPPFLAGS = \
 	-I$(top_srcdir)/source \
 	-I$(top_srcdir)/source/fltkui \
+	-I$(top_srcdir)/source/nes_ntsc \
 	-DDATADIR=\"$(datadir)/nestopia\" \
 	-DDATAROOTDIR=\"$(datarootdir)\" \
 	-DNST_PRAGMA_ONCE \
@@ -17,6 +18,7 @@
 	$(FLTK_CFLAGS)
 nestopia_LDADD = \
 	$(ZLIB_LIBS) \
+	-lnes_ntsc \
 	$(LIBARCHIVE_LIBS) \
 	$(SDL2_LIBS) \
 	$(FLTK_LIBS)
diff -durN nestopia-1.51.0.orig/source/core/NstVideoFilterNtscCfg.cpp nestopia-1.51.0/source/core/NstVideoFilterNtscCfg.cpp
--- nestopia-1.51.0.orig/source/core/NstVideoFilterNtscCfg.cpp	2021-05-19 22:42:36.000000000 +0200
+++ nestopia-1.51.0/source/core/NstVideoFilterNtscCfg.cpp	2021-05-24 12:13:04.369985601 +0200
@@ -34,7 +34,6 @@
 #endif
 
 #define NES_NTSC_NO_BLITTERS
-#include "../nes_ntsc/nes_ntsc.inl"
 
 #ifdef _MSC_VER
 #pragma warning( pop )
diff -durN nestopia-1.51.0.orig/source/core/NstVideoFilterNtsc.hpp nestopia-1.51.0/source/core/NstVideoFilterNtsc.hpp
--- nestopia-1.51.0.orig/source/core/NstVideoFilterNtsc.hpp	2021-05-19 22:42:36.000000000 +0200
+++ nestopia-1.51.0/source/core/NstVideoFilterNtsc.hpp	2021-05-24 12:13:04.370985611 +0200
@@ -25,7 +25,7 @@
 #ifndef NST_VIDEO_FILTER_NTSC_H
 #define NST_VIDEO_FILTER_NTSC_H
 
-#include "../nes_ntsc/nes_ntsc.h"
+#include <nes_ntsc.h>
 
 #ifdef NST_PRAGMA_ONCE
 #pragma once
