diff -durN nestopia-1.52.0.orig/Makefile.am nestopia-1.52.0/Makefile.am
--- nestopia-1.52.0.orig/Makefile.am	2022-12-29 15:22:27.000000000 +0100
+++ nestopia-1.52.0/Makefile.am	2022-12-30 15:12:22.768372316 +0100
@@ -8,6 +8,7 @@
 nestopia_CPPFLAGS = \
 	-I$(top_srcdir)/source \
 	-I$(top_srcdir)/source/fltkui \
+	-I$(top_srcdir)/source/nes_ntsc \
 	-DNST_DATADIR=\"$(datadir)/nestopia\" \
 	-DNST_DATAROOTDIR=\"$(datarootdir)\" \
 	-DNST_PRAGMA_ONCE \
@@ -17,6 +18,7 @@
 	$(FLTK_CFLAGS)
 nestopia_LDADD = \
 	$(ZLIB_LIBS) \
+	-lnes_ntsc \
 	$(LIBARCHIVE_LIBS) \
 	$(SDL2_LIBS) \
 	$(FLTK_LIBS)
diff -durN nestopia-1.52.0.orig/source/core/NstVideoFilterNtscCfg.cpp nestopia-1.52.0/source/core/NstVideoFilterNtscCfg.cpp
--- nestopia-1.52.0.orig/source/core/NstVideoFilterNtscCfg.cpp	2022-12-29 15:22:27.000000000 +0100
+++ nestopia-1.52.0/source/core/NstVideoFilterNtscCfg.cpp	2022-12-30 15:11:26.227205958 +0100
@@ -34,7 +34,6 @@
 #endif
 
 #define NES_NTSC_NO_BLITTERS
-#include "../nes_ntsc/nes_ntsc.inl"
 
 #ifdef _MSC_VER
 #pragma warning( pop )
diff -durN nestopia-1.52.0.orig/source/core/NstVideoFilterNtsc.hpp nestopia-1.52.0/source/core/NstVideoFilterNtsc.hpp
--- nestopia-1.52.0.orig/source/core/NstVideoFilterNtsc.hpp	2022-12-29 15:22:27.000000000 +0100
+++ nestopia-1.52.0/source/core/NstVideoFilterNtsc.hpp	2022-12-30 15:11:26.228205943 +0100
@@ -25,7 +25,7 @@
 #ifndef NST_VIDEO_FILTER_NTSC_H
 #define NST_VIDEO_FILTER_NTSC_H
 
-#include "../nes_ntsc/nes_ntsc.h"
+#include <nes_ntsc.h>
 
 #ifdef NST_PRAGMA_ONCE
 #pragma once
