Description: Preserve externally-provided build flags
Author: Stephen Kitt <skitt@debian.org>

--- a/Makefile
+++ b/Makefile
@@ -2,12 +2,12 @@
 CXX ?= c++
 CXXFLAGS ?= -O3
 CPPFLAGS += -DNST_PRAGMA_ONCE
-CFLAGS = $(shell sdl2-config --cflags)
+CFLAGS += $(shell sdl2-config --cflags)
 
 INCLUDES = -Isource
 WARNINGS = -Wno-write-strings
 
-LDFLAGS = -Wl,--as-needed
+LDFLAGS += -Wl,--as-needed
 LIBS = -lstdc++ -lm -lz
 LIBS += $(shell sdl2-config --libs)
 
